packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -55,25 +66,48 @@ export const TranslationSidebar = ({
         </span>
         <span>{word.lemmaId}</span>
       </div>
-      <div className="overflow-y-auto grow">
-        {lemmaResourcesQuery.isLoading && (
-          <div className="h-full w-full flex items-center justify-center">
-            <LoadingSpinner />
-          </div>
-        )}
-        {lemmaResourcesQuery.isSuccess && lexiconEntry && (
-          <div>
-            <div className="text-lg mb-3 font-bold me-2">
-              {lexiconResource?.resource}
-            </div>
-            <div
-              className="leading-7"
-              dangerouslySetInnerHTML={{
-                __html: DOMPurify.sanitize(lexiconEntry),
-              }}
-            />
-          </div>
-        )}
+      <div className="grow flex flex-col min-h-0">
+        <Tab.Group>
+          <Tab.List className="flex flex-row gap-1 border-b-2 border-slate-800 md:-ms-3 -mx-4 px-3">
+            {tabTitles.map((title) => (
+              <Tab key={title} as={Fragment}>
```

Comment:
The design here is too bold. So a couple suggestions to help it match the tone of the rest of the site.
1. Shrink the border to 1 pixel and change the color to `border-slate-400`. That's a little softer and matches the border color of our input elements.
1. Reduce the padding of the tabs to `px-4 py-1` and remove the `flex-1` so the tabs scale to the size of the content.
1. Use the tab design that removes the bottom border of the selected tab. We can use a spacer element between each tab that has a border, which would then allow each tab to make it's bottom border transparent when selected. The spacer after all of the tabs can have `flex-grow` so that it takes up the remaining space.
1. Add focus outline styles to match the TextInput for accessibility

Reply:
Thanks for the suggestions, I'll try to follow them.

I was having a hard time matching the figma design so I just made something similar to the navigation bar.

Reply:
Styling fixes done.

I pushed two commits with the focus outline. The first uses `border`, but has a bug where the layout shifts because of `border-2`. The second uses `outline`, but the bottom of the outline looks terrible.

Reply:
I think it is fine for now. We could switch to `focus-visible` which would only show that outline if you are using the keyboard to navigate.